DigitalLearningSolutions.Web/Models/Enums/FilterStatus.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace DigitalLearningSolutions.Web.Models.Enums
+{
+    public enum FilterStatus
+    {
+        Default,
+        Warning,
+        Success,
+        Other //This will result in the default blue nhsuk-tag

```

Comment:
Is there a reason we support "Other" at this stage?

Reply:
"Having spoken to Carolyn and David, they agree that the first option ([blue,] grey, red, green) would be preferred. Thanks both." 

Figured I might as well stick it in, but the name is probably bad

Reply:
I thought that [blue] meant the colour of the listed filters, but I will clarify this with Kevin tomorrow morning.. Shouldn't block this though

Reply:
Kevin confirmed that the blue NHS tag can go and we can use blue for applied filters